server/player/state/player_state.py

```diff
@@ -19,24 +19,50 @@
 # along with taserver.  If not, see <http://www.gnu.org/licenses/>.
 #
 import inspect
+from functools import wraps
+
+from player.player import Player
 
 
 class PlayerState:
-    def __init__(self, player):
+    def __init__(self, player: Player):
         self.player = player
 
     def handle_request(self, request):
         methods = [
             func for name, func in inspect.getmembers(self) if
             not name.startswith("__")
```

Comment:
Remove this line because we're not going to find members of PlayerStates that have an attribute handles_packet other than the functions we want.

Reply:
true